src/__tests__/views/viewProfile.test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import ViewProfile from '../../views/profilePage/ViewProfile';
+
+
+describe('get a user profile', () => {
```

Comment:
@Dubby20 we're testing the `ViewProfile` component here, right? If so, please reword your description.